src/serialization/CCLACSets.jl

```diff
@@ -0,0 +1,180 @@
+module CCLACSets
+export read_ccl_acset
+
+using ...ACSetInterface, ...Schemas, ..ACSetSerialization, ...DenseACSets
+
+using ACSets
+
+using JSON3
+using MLStyle
+
+@data AnElement begin
+    ObjectElem(name::String, type::String)
+    HomElem(name::String, dom::String, cod::String)
```

Comment:
Morphisms in models also have types, so this struct needs another field. This will become important when reading, say, a schema from CatColab, where the morphism type will distinguish ordinary morphisms (foreign keys) from data attributes.

Compare with the `ObDecl` and `MorDecl` structs [here](https://github.com/ToposInstitute/CatColab/blob/main/packages/catlog-wasm/src/model.rs).

Also, would suggest choosing your names in a parallel way: here you abbreviate morphism to `Hom` but don't abbreviate `Object`.